docs/README.md

```diff
@@ -4,22 +4,23 @@
 
 | Topic | Description |
 |-------|-------------|
-|**[Getting Started](getting_started.md)** | Set up your development environment.
-|**[Frontend Backend Run Example](frontend_backend_run.md)** | Learn how to run an application locally with tye.
-|**[Frontend Backend Deploy Example](frontend_backend_deploy.md)** | Learn how to deploy an application with tye.
-| **[Adding Redis](redis.md)** | Learn how to add redis for development and deployed to a development cluster.
+|**[Getting Started](getting_started.md)** | Get setup to develop with `tye`.
+|**[Basic Tutorial (run locally)](/docs/tutorials/hello-tye/00_run_locally.md)** | Learn how to run an application locally with `tye`.
```

Comment:
@jkotalik - you good with this? I wanted to move away from seeing `Frontend Backend` in names because it's really not very descriptive.

Reply:
Yep!